Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Base
+FROM python as base
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+ADD / /code/
```

Comment:
Minor, but I would usually recommend `COPY` over `ADD` - in this case they should have identical behaviour, but ADD can handle a couple of extra cases. The standard recommendation is to stick to COPY unless you're explicitly taking advantage of ADD's extra capability, in order to keep things as obvious as possible